Samples/VirtualCamera/README.md

```diff
@@ -0,0 +1,180 @@
+# Virtual Camera Sample #
+### Requirement:
+&emsp;&emsp;Minimum OS version: --TBD-- (current: 10.0.21364.0)

```

Comment:
>Minimum OS version [](http://example.com/codeflow?start=12&length=18)

probably should add a requirement section for developing and compiling this. Seems like you require the WDK to be installed in this version as well as Windows SDK v21364

Reply:
unless WDK dependecy was a mistake :p

Reply:
add WDK reference